apps/nextjs-example/pages/index.tsx

```diff
@@ -340,6 +368,109 @@ function OptionalFunctionality() {
         );
     };
 
+    const onSubmitScript = async () => {
+        const payload = {
+            type: "script_payload",
+            code: {
+                bytecode: DO_NOTHING_SCRIPT
+            },
+            type_arguments: [],
+            arguments: []
+        };
+
+        const response = await signAndSubmitTransaction(payload);
+        try {
+            if (response?.hash === undefined) {
+                throw new Error(`No response given ${response}`)
+            }
+            await aptosClient(network?.name.toLowerCase()).waitForTransaction(response.hash);
+            setSuccessAlertHash(response.hash, network?.name);
+        } catch (error) {
+            console.error(error);
+        }
+        setSuccessAlertMessage(
+            JSON.stringify({signAndSubmitTransaction: response})
+        );
+    };
+
+    // TODO: Let's put this into a cookie or local storage so it only happens once
+    const createFeePayerAccount = async (client: AptosClient) => {
```

Comment:
for the only happens once comment: maybe something like this..?

```typescript
const [feePayerAccount, setFeePayerAccount] = useState<AptosAccount>();

const fundAndSetFeePayer = async(): AptosAccount => {
  const feePayer = new AptosAccount();
  await faucet(network?.name.toLowerCase()).fundAccount(feePayer.address(), 100_000_000));
  setFeePayerAccount(feePayer);
  return feePayer;
}

// create and fund on component mount
useEffect(() => {
  fundAndSetFeePayer();
}, []);

const onSubmitFeePayer = async() => {
  // ...
  const feePayer = feePayerAccount ?? await fundAndSetFeePayer();
  // ...
};
```

Reply:
The idea is to persist the account across sessions, my vote is using `window.localStorage`

Reply:
Let me follow up in a quick PR after this for local storage and all.